### ISSUE TITLE
Remove the duplicate items in the list

### ISSUE THREAD

**Issue by c02y** [tuvistavie/fish-fastdir#1]
In the list like this:

```
1 /home/daniel/.config/fish/fundle/tuvistavie/fish-fastdir
2 /home/daniel
3 /home/daniel/Dropbox
4 /home/daniel/Dropbox/tmp
5 /usr/local
6 /home/daniel
7 /home/daniel/Dropbox
```

there are duplicates, and how can I make all the items unique? Specifically, remove all the old items like 2 and 3 and leave only the new one, of course the numbers in front of the items will be reordered.

**Comment by danhper** [comment]
@c02y Hi, thanks for the feedback!
I agree having duplicated entry is not really helpful.
However, to fix this we we are going to change the implementation a little.
I think the simplest way is to have a helper function that returns a duplicated
free array, and have `__fastdir_dirhist` print it with its index, and
`__fastdir_cd_num` use it to navigate to the relevant directory.
I will give it a shot when I have the time, but do feel free to send a PR :smile:

**Comment by c02y** [comment]
Yeah, I can only solve part of the problem: if the current entry when I use **d** command is already one of the old entries, I can make it remove old ones ( for example, if I'm currently in Num 7 in the list, it will remove 3), but it will not remove 2 since 6 is not the current one).
